docs/_docs/collections.md

```diff
@@ -149,14 +157,90 @@ written out to `<dest>/awesome/some_subdir/some_doc/index.html`.
 </table>
 </div>
 
+## Permalink examples for collections
+
+Depending on how you declare the permalinks in your configuration file, the permalinks and paths get written differently in the `_site` folder. A few examples will help clarify the options.
+
+Let's say your collection is called `apidocs` with `doc1.md` in your collection. `doc1.md` is grouped inside a folder called `mydocs`. Your project's source directory for the collection looks this:
+
+```
+├── \_apidocs
+│   └── mydocs
+│       └── doc1.md
+```
+
+Based on this scenario, here are a few permalink options.
+
+**Permalink configuration 1**: [Nothing configured] <br/>
+**Output**:
+
+```
+├── apidocs
+│   └── mydocs
+│       └── doc1.html
+```
+
+**Permalink configuration 2**: `/:collection/:path/:title:output_ext`  <br/>
+**Output**:
+
+```
+├── apidocs
+│   └── mydocs
+│       └── doc1.html
+```
+
+**Permalink configuration 3**: No collection permalinks configured, but `pretty` configured for pages/posts. <br/>
+**Output**:
+
+```
+├── apidocs
+│   └── mydocs
+│       └── doc1
+│           └── index.html
+```
+
+**Permalink configuration 4**: `/awesome/:path/:title.html`   <br/>
+**Output**:
+
+```
+├── awesome
+│   └── mydocs
+│       └── doc1.html
+```
+
+**Permalink configuration 5**: `/awesome/:path/:title/`  <br/>
+**Output**:
+
+```
+├── awesome
+│   └── mydocs
+│       └── doc1
+│           └── index.html
+```
+
+**Permalink configuration 6**: `/awesome/:title.html` <br/>
+**Output**:  
+
+```
+├── awesome
+│   └── doc1.html
+```
+
+**Permalink configuration 7**: `:title.html`
+**Output**:
+
+```
+├── doc1.html
+```
+
 ## Liquid Attributes
 
 ### Collections
 
-Each collection is accessible via the `site` Liquid variable. For example, if
+Each collection is accessible through the `site` variable. For example, if
```

Comment:
I'm fine with either. Latinates will be very understandable to folks who speak Latin-derived languages, but may prove a barrier for many others. Perhaps removing this sentence altogether and replacing it with:

> Each collection is accessible as a field on the `site` Liquid variable.